Classes/Flowpack/ElasticSearch/ContentRepositoryAdaptor/Eel/ElasticSearchQueryBuilder.php

```diff
@@ -729,4 +730,22 @@ protected function convertHitsToNodes(array $hits)
 
         return array_values($nodes);
     }
+
+    /**
+     * Proxy method to access the public method of the Request object
+     *
+     * This is useful with custom Request type where not wrapper method exist in the QueryBuilder.
+     *
+     * @param string $method
+     * @param array $args
+     * @return $this
+     * @throws Exception
+     */
+    public function __call($method, array $args) {
+        if (!method_exists($this->request, $method)) {
+            throw new Exception(sprintf('Method "%s" does not exist in the current Request object "%s"', $method, get_class($this->request)), 1486763515);
```

Comment:
Shouldn't a `Flowpack\ElasticSearch\ContentRepositoryAdaptor\Exception` be thrown here?